NuGet.config

```
@@ -13,10 +13,11 @@
   </config>
   <packageSources>
     <clear />
-    <add key="dotnet blob feed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="benchmarkdotnet-ci" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
```

is it possible to have two feeds with the same `key`?

```suggestion
    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
    <add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
```

---

The BenchmarkDotNet version that is used in this repo (0.11.3) is available on [nuget.org](https://www.nuget.org/packages/BenchmarkDotNet/0.11.3), so we can just remove the custom BDN feed

https://github.com/dotnet/corefxlab/blob/670f3522f47b3559185bed61da28db18a76bec46/eng/Versions.props#L17